radar/dev/nodejs.yaml

```diff
@@ -4,15 +4,33 @@ blip:
   - date: 2018-03-20
     ring: TRIAL
 description: |
-  _Node.js_ is javascript on the server. It is asynchronous and event driven
-  Javascript runtime built to be scalable and highly concurrent.
+  [_Node.js_](https://nodejs.org/en/) is javascript on the server.
+  It is asynchronous and event driven Javascript runtime
+  built to be scalable and highly concurrent.
+  Node.js is single-threaded and uses an event-loop for processing.
 rationale: |
   Node.js is supported by all cloud provides and is essential in serverless and
   cloud native solutions. It is also a vital part of the front-end stack and a
   candidate for backend-for-frontend services.
+
+  Node.js is particular good of solving these specific types of problems:
+    * Non-blocking operations
+    * Event-driven requests
+    * Data-intensive process
+    * I/O intensive systems
+  The single threaded nature has it limitations
+  and another language might be better to use if you want to solve the following:
```

Comment:
Since it's adopt state maybe we should mention something in regards to https://nodejs.org/api/worker_threads.html which helps solve a lot of the short comings for certain scenarios?

Reply:
The reason being I don't like to encourage going "looking for the next thing" before exhausting the possibilities of existing technologies in Adopt :)

Reply:
I agree, but also we have other languages in the Adopt circle. We should try and use the technology that is best suited.